commands/features.go

```diff
@@ -0,0 +1,54 @@
+package commands
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/ConradIrwin/font/sfnt"
+)
+
+func layoutTable(font *sfnt.Font, tag sfnt.Tag, name string) {
```

Comment:
I’d suggest moving `layoutTable` to the bottom, under `Features` rather than on top.

In Go, it’s common to order code from higher level functions first to lower level helpers second. That way, when reading code, you first encounter higher level, easier to understand code that helps provide context for the helpers. Then, once you're familiar with how a helper is being used and why it’s needed, reading its code is easterly.

Reply:
Thanks for the feedback. Good point. Fixed.